.github/workflows/njsscan-analysis.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow integrates njsscan with GitHub's Code Scanning feature
+# nodejsscan is a static security code scanner that finds insecure code patterns in your Node.js applications
+
+name: njsscan sarif
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '28 3 * * 6'
+
+jobs:
+  njsscan:
```

Comment:
Not sure how useful njsscan will be, doesn't look like it has too much usage as well: https://github.com/ajinabraham/njsscan

Reply:
not sure, but we can easily remove later;
also added codeql in addition.